web-console/monitoring/alertmanager-config/alertmanager-config.md

```diff
@@ -0,0 +1,34 @@
+# Alertmanager Configuration
+
+![alerts list view](img/alerts-list.png)
+- To consolidate all admin-level alerting functions together, `Alerts` and `Silences` pages could be combined onto the same page as tabs with the `Configuration` and `YAMl` pages
+
+![alerting configuation tab](img/config-empty.png)
+- The `Configuration` tab would show some basic grouping information from the default route section of the YAML and a list of receivers for the cluster.
+
+![configuration tab with create menu open showing receiver types](img/config-dropdown.png)
+- Clicking the “create” dropdown would enable users to create a receiver based on an integration type.
+- We could provide the entire list of integration types here, or just a subset as time allows.
+- Users would only be able to create receivers with a single type of integration. More complex receivers could be created from the `YAML` tab.
+
+## Creating the default receiver
+![create receiver page](img/create-default-receiver.png)
```

Comment:
The help text below the From and SMTP fields refer to alerts as "notifications" for the first time here. Should probably be "alerts" right?